src/components/carrossel-oficinas/index.js

```diff
@@ -16,10 +17,16 @@ const WorkshopsCarrossel = () => {
     cms.get('api/workshops/?populate=foto_divulgacao').then((response) => {
       const { data } = response.data
       const workshops = data.map((data) => {
-        return data.attributes
+        if (data) {
+          return {
+            id: data.id,
+            name: data.attributes.nome,
+            image_url: data.attributes.foto_divulgacao.data[0].attributes.url,
```

Comment:
Dado que aqui temos mais controle sobre os nomes de variáveis (ao contrário do CMS), seria ideal seguir as convenções de nomenclatura do Javascript sempre que possível.

```suggestion
            imageUrl: data.attributes.foto_divulgacao.data[0].attributes.url,
```

Reply:
Obrigada pela observação Yasser. Alteração feita.